### ISSUE TITLE
Window driver

### ISSUE THREAD

**Issue by joeblew99** [ValveSoftware/openvr#591]
It seems all the code examples us sdl.
Does this mean that glfw is not supported.

**Comment by vicdxxx** [comment]
Not exactly. The OpenVR or OpenVR Driver SDK are only responsible for connecting your apps with SteamVR( or apps resemble), You can use several languages OpenVR supported to pass your textures.
